docs/source/cc_service.md

```diff
@@ -155,6 +155,7 @@ For example:
 ```json
 {
   "address": "your.chaincode.host.com:9999",
+  "domain": "your.chaincode.host.com",
```

Comment:
Please add in definition above also (line 145).

Reply:
I'm sorry. I'll go right away.